vertx-web/src/main/java/io/vertx/ext/web/handler/impl/StaticHandlerImpl.java

```diff
@@ -507,6 +572,12 @@ public StaticHandler setAlwaysAsyncFS(boolean alwaysAsyncFS) {
     return this;
   }
 
+  @Override
+  public StaticHandler setHttp2PushMapping(Map<String, List<Http2PushMapping>> http2PushMap) {
+    this.http2PushMap = http2PushMap;
```

Comment:
I think we should make a copy when not null, so the handler can assume that it won't be modified